mesonbuild/mtest.py

```diff
@@ -1582,6 +1582,10 @@ def prepare_build(self) -> None:
         if self.options.no_rebuild:
             return
 
+        if (Path(self.options.wd) / 'meson-none-backend').is_file():
+            # nothing to build...
+            self.options.no_rebuild = True
+            return
```

Comment:
likewise, we have the Build object with the exact backend info.

Reply:
Done